src/ALE/MOM_ALE.F90

```diff
@@ -633,7 +631,7 @@ subroutine ALE_build_grid( G, GV, regridCS, remapCS, h, tv, debug, frac_shelf_h
   if (show_call_tree) call callTree_enter("ALE_build_grid(), MOM_ALE.F90")
   use_ice_shelf = .false.
```

Comment:
These 4 lines could be written more simply as ice_shelf = present(frac_shelf_h). Similar comments apply elsewhere now that frac_shelf_h is not a pointer but a simple optional array.